my-contacts/src/ContactEnzyme.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Contact from "./Contact";
+
+
+let contact = shallow(
+    <Contact
+        image=''
+        firstName='John'
+        lastName='Smith'
+        company='My wonderful business'
+        phone='12345'
+        email='john@gmail.com'
+    />
+);
+
+
+describe('Contact details displayed in web view', () => {
```

Comment:
I dnt see setting up a treat provider here. does this pass?

Reply:
Haha, nope!! This was what I was hoping to pair with Zain on around pulling in some of the treat code and trying to provide a default theme. Just got pushed down in priority unfortunately

Reply:
All good